example/src/TopCardsDisplayer.re

```diff
@@ -30,6 +40,31 @@ module SiteStatisticsFragment = [%relay.fragment
 let make = (~siteStatistics as siteStatisticsRef) => {
   let siteStatistics = SiteStatisticsFragment.use(siteStatisticsRef);
 
+  let environment = ReasonRelay.useEnvironmentFromContext();
+  React.useEffect0(() => {
```

Comment:
Will this run every render? I don't remember what 0 does 😅

Reply:
Good point, had to check - we wouldn't want to subscribe on every render now, would we? :)
`useEffect` runs on every render. `useEffect0` runs once on mount and never re-runs. Same as `useEffect1` with an empty deps array, if I am not mistaken.

Reply:
Excellent!